src/applications/hca/actions.js

```diff
@@ -196,3 +205,52 @@ export function setDismissedHCANotification(status, statusEffectiveAt) {
     });
   };
 }
+
+function getResponseError(response) {
+  if (response.errors?.length) {
+    const { code, detail } = response.errors[0];
```

Comment:
@Mottie the API request returns an errors array with status, code and detail, not error as properties of each array element. The code is the EVSS status code. Also, code in used instead of status for this API request in other applications in va.gov. 
https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/form_526/getRatingInfo